tests/EventListener/AddFormatListenerTest.php

```diff
@@ -24,18 +26,18 @@
  */
 class AddFormatListenerTest extends TestCase
 {
-    public function testNoResourceClass()
+    public function testRequestFormatShouldNotBeSet()
     {
-        $request = new Request();
-
+        $requestProphecy = $this->prophesize(Request::class);
```

Comment:
Objects (without related interfaces) from external libraries should not be mocked, especially when its a value object like this one.